.github/workflows/test.yml

```diff
@@ -1,97 +1,83 @@
-name: SmartDeviceLink Unit Tests
+name: SmartDeviceLink Tests
 
 # This workflow is triggered on a push or pull request.
 on:
+    # Runs the workflow when someone pushes to a repository branch, which triggers the push event.
     push:
-      branches:
-        - '*'
-      tags-ignore: # Don't run workflow when a tag is created
-        - v*
```

Comment:
Should these have been removed? Now there's nothing underneath it, which looks odd at least.

Reply:
At least add a comment to explain that nothing needs to be there so it runs on every push if that's what's happening

Reply:
I moved the comment so hopefully it is more clear.